assets/javascripts/modules/checkout/validations.js

```diff
@@ -1,18 +1,24 @@
 define([
     '$',
-    'modules/checkout/form-elements',
-    'modules/checkout/email-check',
+    'utils/text',
+    'modules/checkout/formElements',
+    'modules/checkout/emailCheck',
     'modules/checkout/regex'
-], function ($,
-             form,
-             emailCheck,
-             regex) {
+], function (
+    $,
+    textUtils,
+    form,
+    emailCheck,
+    regex
+) {
     'use strict';
 
-    var ERROR_CLASS = 'form-field--error',
-        EMAIL_ERROR_DEFAULT = 'Please enter a valid Email address',
-        EMAIL_ERROR_TAKEN = 'Your email is already in use! Please sign in or use another email address.',
-        EMAIL_ERROR_NETWORK = 'There\'s been a problem. Please try again later.';
+    var ERROR_CLASS = 'form-field--error';
+    var MESSAGES = {
+        emailInvalid: 'Please enter a valid email address.',
+        emailTaken: 'Your email is already in use. Please sign in or use another email address.',
+        emailFailure: 'There has been a problem. Please try again later.'
```

Comment:
Whoops, good catch meant to add it here https://github.com/guardian/subscriptions-frontend/blob/validation-refactor/assets/javascripts/modules/checkout/validations.js#L87 will sort that.